.github/workflows/issues-chores.yml

```diff
@@ -51,7 +51,29 @@ jobs:
           github-token: ${{ secrets.GH_ISSUES_PRS_CHORES }}
           labeled: Area:QA
 
-      - name: Attach all tickets with label Area:QA to the Security Roadmap Project
+      - name: Attach all tickets with label Area:Security and incident to the Security Roadmap Project
+        uses: actions/add-to-project@v1.0.2
+        id: new-security-incident
+        with:
+          project-url: https://github.com/orgs/Jahia/projects/22
+          github-token: ${{ secrets.GH_JAHIACI_ISSUES_CHORES }}
+          labeled: Area:Security, incident
+          label-operator: AND
+
+      - name: Create a Security issue in the Support Jira
+        if: ${{ steps.new-security-incident.outputs.itemId != '' }}
+        uses: bryannice/gitactions-jira-issue-creation@1.2.1
+        env:
+          JIRA_ACCOUNT_URL: https://jira.jahia.org
+          JIRA_API_TOKEN: ${{ secrets.JIRA_TOKEN }}
+          JIRA_ISSUE_DESCRIPTION: ${{ context.payload.issue.body }}
+          JIRA_ISSUE_SUMMARY: ${{ context.payload.issue.title }} (${{ steps.new-security-incident.outputs.itemId }})
+          JIRA_ISSUE_TYPE: Bug
+          #Vulnerability
+          JIRA_PROJECT: SECBACKLOG
+          JIRA_USERNAME: mdechecchi@jahia.com
```

Comment:
Can you replace this with a secret? Best is an org secret, but for testing you can also add the secret at a repo level.